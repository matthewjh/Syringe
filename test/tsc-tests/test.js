var A = (function () {
    function A() {
    }
    return A;
})();
kgu;
f;
console.log(A);
