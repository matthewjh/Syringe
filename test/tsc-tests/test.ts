class A {
	
}kgu
f
console.log(A);