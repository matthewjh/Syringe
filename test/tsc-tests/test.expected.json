{
  "diagnostics": [
    {
      "message": "Cannot find name 'kgu'.",
      "character": 1,
      "line": 2
    },
    {
      "message": "Cannot find name 'f'.",
      "character": 0,
      "line": 3
    }
  ]
}